src/Playwright/API/Generated/IBrowserType.cs

```diff
@@ -69,6 +70,20 @@ public partial interface IBrowserType
         /// <summary><para>A path where Playwright expects to find a bundled browser executable.</para></summary>
         string ExecutablePath { get; }
 
+        /// <summary>
+        /// <para>Returns the remote browser instance.</para>
+        /// <para>
+        /// Connect the remote browser with <paramref name="wsEndpoint"/>
+        /// and returns the remote browser.
+        /// </para>
+        /// <code>
+        /// var browser = await playwright.Chromium.ConnectAsync("ws://url");
+        /// </code>
+        /// </summary>
+        /// <param name="wsEndpoint">WebSocket Endpoing</param>
+        /// /// <param name="options">Connect options</param>
+        Task<IBrowser> ConnectAsync(string wsEndpoint, BrowserTypeConnectOptions? options = default);
```

Comment:
Do we have a PR upstream generating this new API?

Reply:
https://github.com/microsoft/playwright/pull/7469